help/tip-of-the-day/tips_en.txt

```diff
@@ -1,16 +1,15 @@
-While editing a map, press Tab to switch between the edit tool and the drawing tools quickly.
-A map with few well chosen features will give a much better map than a map cluttered with many insignificant features.   Eduard Imhof
+While editing a map, press <b>Tab</b> to switch between the edit tool and the drawing tools quickly.
+A map with few well chosen features will give a much better map than a map cluttered with many insignificant features.<br/><br/>Eduard Imhof
 Features that are not important for a competitor taking part in a sprint orienteering event should not be mapped. Examples of this are waste baskets, fire hydrants, parking meters and individual street lights as they clutter the map with unhelpful detail.
 In the end, it is the mapmaker's task to produce precise and legible orienteering maps by applying the specifications and generalisation rules, such as selection, simplification and exaggeration.
 Mapping multilevel structures (bridges, underpasses or underground buildings) is confusing. Generally only the main "running" level should be shown. However, underground passages (e.g. underpasses, lighted tunnels) or overpasses (e.g. bridges), which are important for competitors should be mapped.
 All line widths and symbol dimensions must be kept strictly to their specified value. Certain minimum dimensions must also be observed. These are based on both printing technology and the need for legibility.
-Minimum gap between two line symbols of the same colour, in brown or black: 0.15 mm, in blue: 0.25 mm
-Minimum gap between line symbols and area symbols of the same colour, in black: 0.15 mm
-Shortest dotted line: at least two dots
-Shortest dashed line: at least two dashes
-Smallest area enclosed by a dotted line: 1.5 mm (diameter) with 5 dots
-Smallest area of colour:<br/>Blue, green, grey or yellow full colour: 0.5 mm2; <br/>Black dot screen: 0.5 mm2; <br/>Blue, brown, green or yellow dot screen: 1.0 mm2
-All features plotting smaller than the minimum dimensions must be either exaggerated or omitted, depending on whether or not they are of significance to the competitor.
+Minimum gap between two line symbols of the same colour, in brown or black: 0.15 mm, in blue: 0.25 mm.
+Minimum gap between line symbols and area symbols of the same colour, in black: 0.15 mm.
+Shortest dotted line: at least two dots.
+Shortest dashed line: at least two dashes.
+Smallest area enclosed by a dotted line: 1.5 mm (diameter) with 5 dots.
+Smallest area of colour:<ul><li>Blue, green, grey or yellow full colour: 0.5 mm<sup>2</sup></li><li>Black dot screen: 0.5 mm<sup>2</sup></li><li>Blue, brown, green or yellow dot screen: 1.0 mm<sup>2</sup></li></ul>All features plotting smaller than the minimum dimensions must be either exaggerated or omitted, depending on whether or not they are of significance to the competitor.
```

Comment:
This used to be two separate lines. The second sentence applies not only to "areas of colour" but also to the lines before IMO.